www/src/data/StarterShowcase/generatedGithubData/gatsby-advanced-blog.json

```diff
@@ -1 +1,195 @@
hub.com/repos/wonism/gatsby-advanced-blog/git/tags{/sha}","git_refs_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/git/refs{/sha}","trees_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/git/trees{/sha}","statuses_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/statuses/{sha}","languages_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/languages","stargazers_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/stargazers","contributors_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/contributors","subscribers_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/subscribers","subscription_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/subscription","commits_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/commits{/sha}","git_commits_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/git/commits{/sha}","comments_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/comments{/number}","issue_comment_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/issues/comments{/number}","contents_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/contents/{+path}","compare_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/compare/{base}...{head}","merges_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/merges","archive_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/{archive_format}{/ref}","downloads_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/downloads","issues_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/issues{/number}","pulls_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/pulls{/number}","milestones_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/milestones{/number}","notifications_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/notifications{?since,all,participating}","labels_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/labels{/name}","releases_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/releases{/id}","deployments_url":"https://api.github.com/repos/wonism/gatsby-advanced-blog/deployments","created_at":"2018-03-28T16:01:19Z","updated_at":"2018-06-22T15:39:28Z","pushed_at":"2018-04-22T08:53:31Z","git_url":"git://github.com/wonism/gatsby-advanced-blog.git","ssh_url":"git@github.com:wonism/gatsby-advanced-blog.git","clone_url":"https://github.com/wonism/gatsby-advanced-blog.git","svn_url":"https://github.com/wonism/gatsby-advanced-blog","homepage":"https://wonism.github.io/","size":2722,"stargazers_count":20,"watchers_count":20,"language":"JavaScript","has_issues":true,"has_projects":true,"has_downloads":true,"has_wiki":true,"has_pages":false,"forks_count":6,"mirror_url":null,"archived":false,"open_issues_count":1,"license":null,"forks":6,"open_issues":1,"watchers":20,"default_branch":"master","network_count":6,"subscribers_count":4}}
```

Comment:
we might want to ignore those.. or just format them as well if it's not an issue.